plugins/vpc-branch-eni/plugin/commands.go

```diff
@@ -57,14 +58,28 @@ func (plugin *Plugin) Add(args *cniSkel.CmdArgs) error {
 	}
 
 	// Lookup the user ID for TAP link ownership.
-	var uid int
+	var uid, gid int
 	if netConfig.InterfaceType == config.IfTypeTAP {
-		uid, err = plugin.LookupUser(netConfig.UserName)
-		if err != nil {
-			log.Errorf("Failed to lookup user %s: %v.", netConfig.UserName, err)
-			return err
+		if netConfig.Uid != "" && netConfig.Gid != "" {
```

Comment:
What we are doing here is configuration validation, whether UID and GID are valid integers. Therefore this logic should be done in netconfig.go, together with the rest of the validation. Can we please move the strconv calls to netconfig.go line 168?

Reply:
Same as previous comment, let me how you think about it then I will try to refactor this if required.